proc/syscall_windows.go

```diff
@@ -22,8 +22,53 @@ type _THREAD_BASIC_INFORMATION struct {
 	BasePriority   int32
 }
 
+type _CREATE_PROCESS_DEBUG_INFO struct {
+	File                syscall.Handle
+	Process             syscall.Handle
+	Thread              syscall.Handle
+	BaseOfImage         uintptr
+	DebugInfoFileOffset uint16
+	DebugInfoSize       uint16
+	ThreadLocalBase     uintptr
+	StartAddress        uintptr
+	ImageName           *byte
```

Comment:
From the doco: "This parameter may be NULL, or it may contain the address of a string pointer in the address space of the process being debugged.", so maybe we can convert this into Go string (from ansii or utf16 depending on the fUnicode value). I don't see why you say that it is "in the address space of the debuggee". Where in the doco it says that? I think everything we see is in caller's address space.